test/TestProjects/Customizations-TypeSpec/src/Customizations/RenamedModel.cs

```diff
@@ -6,5 +6,9 @@
 namespace CustomizationsInCadl.Models
 {
     [CodeGenModel("ModelToRename")]
-    public partial class RenamedModel { }
+    public partial class RenamedModel
+    {
+        /// <summary> Optional int. </summary>
```

Comment:
Move this property here to mimic the scenario as described in [this issue](https://github.com/Azure/autorest.csharp/issues/3474)